frontend/appflowy_flutter/lib/plugins/database_view/tar_bar/tar_bar_add_button.dart

```diff
@@ -156,16 +156,16 @@ enum AddButtonAction {
     }
   }
 
-  String get iconName {
+  FlowySvgData get icon {
     switch (this) {
       case AddButtonAction.board:
-        return 'editor/board';
+        return FlowySvgs.board_editor;
       case AddButtonAction.calendar:
-        return "editor/grid";
+        return FlowySvgs.grid_editor;
       case AddButtonAction.grid:
-        return "editor/grid";
+        return FlowySvgs.grid_editor;
       default:
-        return "";
+        return const FlowySvgData('');
```

Comment:
Better default value?

Reply:
Is it possible to remove it completely? It is an exhaustive search on the enum after all

Reply:
oh duh, that's right haha